### ISSUE TITLE
Loss became NaN when try to train with kitti

### ISSUE THREAD

**Issue by jlyw1017** [bonlime/keras-deeplab-v3-plus#66]
Hey bonlime,
Thanks a lot for your work!
I'm having a problem during retraining of this network using kitti.
The loss became Nan after a few epoch.
Have you met something like that before?
Sofar I have tried different optimizer and very small lr, but nothing works.
Thank you!

here are my codes
    def pack_into_tensor(file_input, file_label):
        img_array_input = np.array([sp.imresize(sp.imread(i), (375, 1242)) for i in file_input])
        img_array_label = np.array([sp.imresize(sp.imread(i), (375, 1242), interp='nearest') for i in file_label])
        img_array_label = np.eye(35)[img_array_label]
    return img_array_input, img_array_label

    def read_data(path_raw, path_label):
    '''    Read files from kitti folder and return tensors as inputs    '''
    # get files
        data_input = glob.glob(path_raw)
        data_input.sort()
        label_input = glob.glob(path_label)
        label_input.sort()
    # put file into tensor
        image_input_tensor, label_input_tensor = pack_into_tensor(data_input, label_input)
        return image_input_tensor, label_input_tensor

    path_image = os.path.join('/home/user/datasets/kitti', "training/image_2", "000*_10.png")
    path_label = os.path.join('/home/user/datasets/kitti', "training/semantic", "000*_10.png")

    input_tensor, label_tensor = read_data(path_image, path_label)
    deeplab_model = Deeplabv3(input_shape=(375, 1242, 3), classes=35, backbone='mobilenetv2')
    opter = keras.optimizers.Adagrad(lr=0.001, epsilon=None, decay=0.0)
    deeplab_model.compile(optimizer=opter, loss='categorical_crossentropy')
    deeplab_model.fit(input_tensor, label_tensor, validation_split=0.1, shuffle=True, nb_epoch=10, Batch_size=4,
                 callbacks=[TensorBoard(log_dir='/home/user/Semantic_Deeplabv3_Kitti/mytensorboard', write_images= 1)])
    deeplab_model.save_weights('/home/user/Semantic_Deeplabv3_Kitti/kitti_weights_pretrainedcoco.h5')

**Comment by bonlime** [comment]
Hi, I have encountered this problem but don't remember how I solved it. 
You can check this https://github.com/bonlime/keras-deeplab-v3-plus/issues/63 maybe there would be some useful hints which you can use.
